plugins/@grouparoo/google-sheets/src/migrations/000001-plugin-google-sheets-aggregationMethod.ts

```diff
@@ -0,0 +1,57 @@
+import Sequelize from "sequelize";
+import crypto from "crypto";
+
+export default {
+  up: async (queryInterface: Sequelize.QueryInterface) => {
+    const allProperties = [];
+    const newOptions = [];
+    const [sources] = await queryInterface.sequelize.query(
+      `SELECT * FROM "sources" WHERE type = 'google-sheet-import'`
+    );
+    console.log(sources);
+    for (const source of sources) {
+      const [properties] = await queryInterface.sequelize.query(
+        `SELECT * FROM "properties" WHERE "sourceId" = '${source["id"]}'`
+      );
+      allProperties.push(...properties);
+    }
+
+    for (const property of allProperties) {
+      newOptions.push({
+        id: `opt_${crypto.randomBytes(16).toString("hex")}`,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+        ownerId: property["id"],
+        ownerType: "property",
+        key: "aggregationMethod",
+        value: "exact",
+        type: "string",
+      });
```

Comment:
I'm a little worries about collisions in the future because these aren't really UUIDs here. Perhaps use the UUID module to generate these IDs